content/en/tracing/generate_metrics/_index.md

```diff
@@ -0,0 +1,87 @@
+
+| Reason                        | Custom Metrics Generated from Spans                   | Retention Filters                           |
+| -------------------------------------- | -------------------------------------- | --------------------------------- |
+| Retention Period                     | 15 months                    | 15 days             |
+| Anomaly Detection                           | Create an [Anomaly Monitor][3] based on generated metrics.                            | Use Analytics to compare behavior over the past 15 days, and drill into complete traces to investigate root cause.                         |
+| Investigation of matching traces with full context                          | N/A - Custom Metrics will not result in any retention of associated traces.                            | Keep exactly the traces relevant to your business context with [retention filters][1].                            |
+| Granularity of behavior                           | Create custom metrics for important endpoints or other low-cardinality groups.                        | Use [Search and Analytics][4] for specific endpoints, or use the 'Group By' option in [Analytics][5].                    |
+| Forecasting or complex mathematics                          | Create a [Forecast monitor][6] based on generated metrics.                          |   N/A                            |
+
+To generate metrics from span, navigate from the [APM Setup and Configuration][7] page to [Generate Metrics][8] on the top of the page, and click the “+ New Metric” button.
+
+<br>
+
+{{< img src="tracing/span_to_metrics/GenerateMetrics.png" style="width:100%;" alt="Generate metrics from ingested spans" >}}
+
+
+### Creating a New Span-based Metric
+
+{{< img src="tracing/span_to_metrics/createspantometrics.png" style="width:100%;" alt="How to create a metric" >}}
+
+1. Define the metric query: Start by adding a query for filtering to your required dataset. The [query syntax][9] is the same as APM Search and Analytics.
+
+1. Define the field you want to track: Select * to generate a count of all spans matching your query or enter an attribute (e.g., `@cassandra_row_count`) to aggregate a numeric value and create its corresponding count, min, max, sum, and avg aggregated metrics. If the attribute type is a measure, the value of the metric is the value of the span attribute.
+
+1. Specify the group-by dimension: By default, metrics generated from spans will not have any tags unless explicitly added. Any attribute or tag that exists in your spans can be used to create metric Tags.
+
+1. Check the Live Analytics and Search Query preview: You can view the impact of your query in real-time on the data visualization, and the matching spans considered for your query in a live preview.
+
+1. Name your metric: Metric names must follow the [metric naming convention][10]. Metric names beginning with `trace.*` are not permitted and will not be saved.
```

Comment:
```suggestion
1. **Name your metric:** Metric names must follow the [metric naming convention][10]. Metric names that start with `trace.*` are not permitted and will not be saved.
```
Do you mean "that start with `trace.` maybe?

Reply:
Yeah I think it's okay to have it as `trace.*` since we refer to them as that in a lot of places and it's the wildcard.  But I like the reword!